Common/src/Stream/Worker/WorkerStreamHandler.cs

```diff
@@ -115,7 +115,9 @@ public async Task Init(CancellationToken cancellationToken)
 
     var e = new ArmoniKException("Could not get grpc channel");
     logger_.LogError(e,
+#pragma warning disable CA2254 // string.Empty is constant
                      string.Empty);
+#pragma warning restore CA2254
```

Comment:
You also could use `""` to avoid the warning. Not sure which is better. Maybe just actually add a message.